tests/tacacs/test_authorization.py

```diff
@@ -32,6 +32,22 @@ def ssh_connect_remote(remote_ip, remote_username, remote_password):
     return ssh
 
 
+def ssh_connect_remote_retry(remote_ip, remote_username, remote_password, duthost):
+    retry_count = 3
+    while retry_count > 0:
+        try:
+            return ssh_connect_remote(remote_ip, remote_username, remote_password)
+        except paramiko.ssh_exception.AuthenticationException as e:
+            logger.info("Paramiko SSH connect failed with authentication: " + repr(e))
+
+            # get syslog for debug
```

Comment:
Why do you dump syslog in test case?

Reply:
The ssh connection randomly failed and we don't know the failed reason. so dump device side syslog for debug.